cirq-core/cirq/experiments/z_phase_calibration_test.py

```diff
@@ -205,3 +206,33 @@ def test_plot_z_phase_calibration_result():
     np.testing.assert_allclose(axes[1].lines[0].get_xdata().astype(float), [1, 2, 3])
     np.testing.assert_allclose(axes[1].lines[0].get_ydata().astype(float), [0.6, 0.4, 0.1])
     np.testing.assert_allclose(axes[1].lines[1].get_ydata().astype(float), [0.7, 0.77, 0.8])
+
+
+@pytest.mark.parametrize('angles', np.random.random((10, 10)))
+def test_transform_circuit(angles):
+    theta, phi = angles[:2]
```

Comment:
Nit - perhaps normalize to possible range?

Reply:
done